CHANGELOG.md

```diff
@@ -70,7 +70,7 @@ In addition to the style changes for Venia, release 7.0.0 also adds a new home p
 The content of this CMS page is defined in the Magento Admin using the Page Builder extension.
 It showcases Page Builder content types such as Sliders, Banners, and Gallery Items.
 
-Check it out: https://develop.pwa-venia.com/venia-new-home
+This new page replaces the old home page content in Venia.
```

Comment:
We should provide a hotlink to the new page.

Reply:
We can provide a link to the 7.0 staging server but that is only valid until the release. Post release, we will have to change it back to the develop link.

Reply:
Resolve by: 98b269c